### PR TITLE
Merge 03-11-add_config_changes into main

### PR DESCRIPTION
**Pull request by cyzhzhd**

### CODE REVIEW

**Comment by cyzhzhd** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/cyzhzhd/graphite/1" target="_blank">Graphite</a>  (this PR)
    * **PR #2** <a href="https://app.graphite.dev/github/pr/cyzhzhd/graphite/2" target="_blank">Graphite</a> 
      * **PR #3** <a href="https://app.graphite.dev/github/pr/cyzhzhd/graphite/3" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

**Comment by jaeholee2212** [comment]
Nice shot!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # graphite
 first change
+second change
```
Comment (jaeholee2212) [inline README.md:3]: This one should be third change, not 2nd.

**Review by jaeholee2212 — CHANGES_REQUESTED** [review]
See inline comments

**Pull request merged**